installer/roles/create-mobile-client-crd/files/mobile-client-crd.yaml

```diff
@@ -26,4 +26,10 @@ spec:
               pattern: '(\w{8}-\w{4}-\w{4}-\w{4}-\w{11})'
             name:
               type: string
-              pattern: '(\w-)'
\ No newline at end of file
+              pattern: '(\w-)'
+            appIdentifier:
+              type: string
+            excludedServices:
+              type: array
+              items:
+                type: string
```

Comment:
this might be a stupid question, but should we look to include a pattern-match for these new params also, assuming they have fairly strict formats?

Reply:
I did consider it, but as the names of these are reasonably open ended, I wasn't sure how much value I would add with something like `[a-z0-9\-]+`, WDYT?